jira/validators/fixversion.groovy

```diff
@@ -0,0 +1,26 @@
+import org.apache.log4j.Level
+
+log.setLevel(Level.DEBUG)
+log.info("Start FixVersion Validation")
+
+// If resolution is not Fixed or Done return always true
+if (!(issue.resolutionObject.name == "Fixed" || issue.resolutionObject.name == "Done")) {
+    log.info("Status is not Fixed or Done (true)")
+    return true
+}
+
+// Check that we have FixedVersions
+if (issue.getFixVersions().isEmpty()) {
+    log.info("FixedVersions is empty (false)")
+    return false
+}
+
+// Check that all AffectedVersions are Fixed
+def versions = issue.getAffectedVersions().minus(issue.getFixVersions())
+if (versions.isEmpty()) {
+    log.info("All AffectedVersions are fixed (true)")
```

Comment:
This may be better:
log.info("All affectedVersions equal fixedVersions (true)")